pages/common/iotop.md

```diff
@@ -30,3 +30,7 @@
 - Show accumulated I/O instead of bandwidth:
 
 `sudo iotop --accumulated`
+
+- Turn on non-interactive mode,and only show processes or threads actually doing I/O
+
+`iotop  -n1 -b -o`
```

Comment:
We use long options where possible and the token syntax `{{ }}` is for highlighting arguments.
```suggestion
`iotop  --iter {{1}} --batch --only`
```